md4c/md4c.c

```diff
@@ -3236,8 +3238,8 @@ md_collect_marks(MD_CTX* ctx, const MD_LINE* lines, int n_lines, int table_mode)
                 continue;
             }
 
-            /* A potential table cell boundary. */
-            if(table_mode  &&  ch == _T('|')) {
+            /* A potential table cell boundary or wiki link label delimiter. */
+            if((table_mode || ctx->parser.flags & MD_FLAG_WIKILINKS) && ch == _T('|')) {
```

Comment:
Noticed this issue in the code: Bit operator `&` has higher priority then `||` so the part of the condition in the brackets is equivalent to `(table_mode | flags) & WIKILINKS`. It should be `table_mode | (flags & WIKILINKS)`.

Reply:
Sorry. Not true, it's `==` which usually makes troubles. But I would prefer if you add the parenthesis there anyway, please.